app/3.0/docs/devguide/custom-elements.md

```diff
@@ -241,22 +263,27 @@ The custom element state isn't exposed as a property, but you can style elements
 whether they're defined or undefined.
 
 Elements in the "custom" and "uncustomized" state are considered "defined". In CSS you can use the
-`:defined` pseudo-class selector to target elements that are defined. You can use this to provide
-placeholder styles for elements before they're upgraded:
+`:defined` pseudo-class selector to target elements that are defined. **In native shadow DOM**,
+you can use this to provide placeholder styles for elements before they're upgraded:
 
 ```
 my-element:not(:defined) {
   background-color: blue;
 }
 ```
 
+**`:defined` is not supported by the Custom Elements polyfill.** See the [documentation on styling](style-shadow-dom#style-undefined-elements) for a workaround.
```

Comment:
This link doesn't seem to exist anymore?

Reply:
I put it back in https://github.com/Polymer/docs/pull/2476/commits/d9d4ff6473c85393def6d755d7745b661f3d9d63